app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,290 @@
+        }
+    }
+
+    /**
+     * Insert invitation into database
+     *
+     * @param  $request
+     */
+    public function createInvitation(Request $request)
+    {
+        try
+        {
+            Invite::create([
+                'username'  => $request->username,
+                'role_id'  => $request->user_role,
+                'token'   => $request->_token
+            ]);
+            $this->response =
+            [
+                'message' => 'Invitation created',
+                'status_code' => 201
+            ];
+        } catch (QueryException $e) {
+            $this->response =
+            [
+                'message' => 'Invitation already sent',
+                'status_code' => 400
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Check if user exist and Insert Invitation details
+     * into the database
+     *
+     * @param  $request
+     */
+    public function insertInvite(Request $request)
+    {
+        if ( $this->checkIfUserExist($request) )
+        {
+            $this->response = $this->createInvitation($request);
+        }
+        else
+        {
+            $this->response =
+            [
+                'message' => 'User does not exist',
+                'status_code' => 400
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Process invitation details into invites table
+     *
+     * @param  Request $request
+     */
+    public function createInvite(Request $request)
+    {
+        $createInvite = $this->insertInvite($request);
+
+        if ($createInvite['status_code'] == 201)
+        {
+            $this->response = $this->getInviteeEmail($request);
+        }
+        else
+        {
+            $this->response =
+            [
+                'message' => $createInvite['message'],
+                'status_code' => $createInvite['status_code']
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Get Invitee email and process mail
+     *
+     * @param  Request $request
+     */
+    public function getInviteeEmail(Request $request)
+    {
+        $email = User::where('username', $request->username)->first();
+
+        if ($email)
+        {
+            $this->response = $this->sendMail($request, $email);
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Send Invitation mail to selected user
+     *
+     * @param  $username
+     * @param  $token
+     * @param  $email
+     *
+     * @return integer
+     */
+    public function sendMail(Request $request, $email)
+    {
+        $mailSent = $this->mail->send('emails.adminInvite', ['username' => $request->username, 'token' => $request->_token], function ($message) use ($email) {
+                $message->from(getenv('SENDER_ADDRESS'), getenv('SENDER_NAME'));
+                $message->to($email->email)->subject('Suyabay Invitation');
+            });
+
+        if ($mailSent)
```

Comment:
The opening brace should on the same line as the `if` . Check out the PSR-2 guide [http://www.php-fig.org/psr/psr-2/](http://www.php-fig.org/psr/psr-2/)